index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header-section">
+        <p class="logo">My logo </p> 
+
+<nav class="bttn">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+</nav>
+    </header>
+    <br> <br>
+    <section  >
+        <img class="background-image"src="i-bttn/background-shapes/header-shapes mobile.png" alt="">
```

Comment:
- [OPTIONAL] Kindly make sure the background image of this page is inserted on the page by using the CSS `background` properties which is the professional way to show the background image on te page.

_**The first review missed to mention this but it is an important part of the project, I highly advise you to implement it before moving on another project**_